types/react-navigation/index.d.ts

```diff
@@ -262,7 +263,7 @@ export type NavigationStackScreenOptions = NavigationScreenOptions & {
   headerTitleStyle?: Style,
   headerTintColor?: string,
   headerLeft?: React.ReactElement<any>,
-  headerBackTitle?: string,
+  headerBackTitle?: string | null,
```

Comment:
This is typed as just `string` in `TypeDefinition.js`. Is there a specific reason to make this allow `null` too? Just wondering, since this introduces a difference from the official Flow typings.